source/winConsoleHandler.py

```diff
@@ -46,7 +46,6 @@
 COMMON_LVB_UNDERSCORE=0x8000
 
 
-

```

Comment:
Looks like removing this blank line is the only change in this file. This change can probably be removed.